src/pyrad_proc/pyrad/io/mxpol_config.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
```

Comment:
I do not think it is necessary to have the config file separated from the reader